tests/acceptance/29_simulate_mode/files_promises.cf.sub

```diff
@@ -0,0 +1,141 @@
+bundle common simulation_files_promises
+{
+  vars:
+    "inputs" slist => { "$(this.promise_dirname)/../default.cf.sub" };
+}
+
+body common control
+{
+  inputs => { "@(simulation_files_promises.inputs)" };
+}
+
+bundle agent init
```

Comment:
hummm your creating files in init? Since this is run as a sub-agent with --simulate=manifest, it seems like there shouldn't be any init bundle.

Reply:
oh, this sub agent executes ANOTHER sub agent.

Reply:
no it doesnt;

Reply:
Right. I had a main bundle in the sub policy which called both init and test. Refactored now. :+1: